### PR TITLE
update redis-setup.sh

### PR DESCRIPTION
**Pull request by sandeepsajan0**
Now if Redis is installed, then it will only run the server not install Redis and docker every time.

### CODE REVIEW

File: redis_setup.sh [hunk 1]
```diff
@@ -15,10 +18,19 @@ sudo add-apt-repository    "deb [arch=amd64] https://download.docker.com/linux/u
 sudo apt-get update
 sudo apt-get install docker-ce
 
+else
+    echo "yesss exist"
```
Comment (xadahiya) [inline redis_setup.sh]: Fix these messages. Please use something professional.

Comment (sandeepsajan0) [reply redis_setup.sh]: ohh sorry! I was using it for checking.

File: redis_setup.sh [hunk 2]
```diff
@@ -1,5 +1,8 @@
 
 #It will remove all the previous docker engine
+docker -v
+if [ echo "$?" = "127" ]
```
Comment (xadahiya) [inline redis_setup.sh]: What is this?

File: redis_setup.sh (hunk 1 above)
Comment (xadahiya) [reply redis_setup.sh]:
```
./redis_setup.sh 
Docker version 18.06.0-ce, build 0ffa825
./redis_setup.sh: line 4: [: too many arguments
Docker is already installed
``` 
Fix this error message

File: redis_setup.sh (hunk 2 above)
Comment (sandeepsajan0) [reply redis_setup.sh]: if there will be no docker in machine, the command `docker -v` will throw error `127`. But if docker is already present, the command will throw  `0`. 
So, it is just a check for docker. And same I have done in line 26, for check `redislabs/redisgraph` docker image.

Comment (xadahiya) [reply redis_setup.sh]: > #It will remove all the previous docker engine
Update this comment to explain this then we can merge this.

**Pull request merged**